### PR TITLE
Bug fixes trevor

### PR DESCRIPTION
**Pull request by LevoraShelby**
Resolved the tasks I claimed on the issue tracker.

### CODE REVIEW

**Review by yue-su — COMMENTED** [review]
Looks good!

File: client/src/components/Header.jsx [hunk 1]
```diff
@@ -9,6 +9,7 @@ import {
   Popover,
   MenuItem,
   Badge,
+  Link as MuiLink,
```
Comment (johnsciutto) [inline client/src/components/Header.jsx:12]: It's a good idea to rename it so it doesn't conflict with the `react-router-dom` link, but are you using it?

**Review by johnsciutto — COMMENTED** [review]
Looks good to me 🙂

File: client/src/components/Header.jsx (hunk 1 above)
Comment (LevoraShelby) [reply client/src/components/Header.jsx:12]: Shouldn't be there anymore, but I was renaming it because we were also importing `Link` from react-router.

**Pull request merged**